service/src/main/java/datawave/microservice/authorization/OAuthControllerV2.java

```diff
@@ -0,0 +1,85 @@
+package datawave.microservice.authorization;
+
+import datawave.microservice.authorization.user.DatawaveUserDetails;
+import datawave.security.authorization.oauth.OAuthTokenResponse;
+import datawave.security.authorization.oauth.OAuthUserInfo;
+import io.swagger.v3.oas.annotations.ExternalDocumentation;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Collection;
+
+/**
+ * Presents the REST operations for the authorization service to implement the OAuth2 code flow.
+ */
+@Tag(name = "OAuth Operations /v2",
+                externalDocs = @ExternalDocumentation(description = "Authorization Service Documentation",
+                                url = "https://github.com/NationalSecurityAgency/datawave-authorization-service"))
+@RestController
+@RequestMapping(path = "/v2/oauth", produces = MediaType.APPLICATION_JSON_VALUE)
+public class OAuthControllerV2 {
+    
+    private final OAuthOperationsV2 oauthOperations;
+    
+    @Autowired
+    public OAuthControllerV2(OAuthOperationsV2 oauthOperations) {
+        this.oauthOperations = oauthOperations;
+    }
+    
+    @Operation(summary = "Authorizes the calling user to produce a JWT value",
```

Comment:
This doesn't seem to produce a value.

Reply:
httpServletResponse is where the response content is set.  Again, this isn't a new method on my part - just moving things around a bit.  Bill can speak to this.

Reply:
ok